panels/models.py

```diff
@@ -57,7 +57,7 @@ def __getattr__(self, name):
 @Session.model_mixin
 class Attendee:
     assigned_panelists = relationship('AssignedPanelist', backref='attendee')
-    panel_applicant = relationship('PanelApplicant', backref='attendee', uselist=False)
+    panel_applicants = relationship('PanelApplicant', backref='attendee')
```

Comment:
Surprisingly, this property was not referenced anywhere, in any of our plugins! So this is the only line that needed to change 😄